### PR TITLE
Task/hw 57464 GitHub.io page i os core

### PR DESCRIPTION
**Pull request by sarun-altmtrk**

### CODE REVIEW

File: Sources/Hyperwallet.swift [hunk 1]
```diff
@@ -111,6 +111,24 @@ public final class Hyperwallet: NSObject {
...
+    /// This function will request a new authentication token via `HyperwalletAuthenticationTokenProvider`
+    /// if the current one is expired or is about to expire.
+    ///
+    /// - Parameters:
+    ///   - account: the `HyperwalletBankAccount` to be created
+    ///   - completion: the callback handler of responses from the Hyperwallet platform
+    public func createBankAccountObjectiveC(account: HyperwalletBankAccount,
+                                            completion: @escaping (HyperwalletBankAccount?, Error?) -> Void) {
+        return createBankAccount(account: account, completion: completion)
+    }
```
Comment (jsaini-hw) [inline Sources/Hyperwallet.swift]: this method has been deleted. Kindly pull latest from development branch

File: .gitignore [hunk 2]
```diff
@@ -8,7 +8,6 @@
 ## Build generated
 build/
 DerivedData/
-docs/
```
Comment (jsaini-hw) [inline .gitignore]: Add it back

File: .travis.yml [hunk 3]
```diff
@@ -47,3 +51,11 @@ script: |
...
 
 after_success: slather
+deploy:
+    provider: pages
+    skip-cleanup: true
+    github-token: $CI_USER_TOKEN  # Set in the settings page of your repository, as a secure variable
+    local-dir: docs
+    target-branch: gh-pages
+    on:
+       all_branches: true
```
Comment (jsaini-hw) [inline .travis.yml]: change it to master

File: .gitignore (hunk 2 above)
Comment (sarun-altmtrk) [reply .gitignore]: Added it.

File: .travis.yml (hunk 3 above)
Comment (sarun-altmtrk) [reply .travis.yml]: Updated.

File: .travis.yml [hunk 4]
```diff
@@ -45,5 +47,16 @@ script: |
     -configuration Debug ONLY_ACTIVE_ARCH=YES ENABLE_TESTABILITY=YES CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO
   # Lint - Add `--strict` to fail for warning and violation
   swiftlint lint --reporter json
+  # Making document for core SDK
+  - make docs
 
 after_success: slather
+    - if [ $TRAVIS_BRANCH = 'master' ] && [ $TRAVIS_PULL_REQUEST = 'false' ]; then sh ./scripts/travisPublishDocs.sh; fi
```
Comment (jsaini-hw) [inline .travis.yml]: where's the travisPublishDocs.sh?

File: .travis.yml [hunk 5]
```diff
@@ -45,5 +47,16 @@ script: |
...
+deploy:
+    provider: pages
+    skip-cleanup: true
+    github-token: $CI_USER_TOKEN  # Set in the settings page of your repository, as a secure variable
+    local-dir: docs
+    target-branch: gh-pages
+    on:
+       branch: master
```
Comment (jsaini-hw) [inline .travis.yml]: If you are using ./scripts/travisPublishDocs.sh; to publish documents then why do you need this deploy section?

File: scripts/travisPublishDocs.sh [hunk 6]
```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+# Set identity
+git config --global user.email "travis@travis-ci.org"
```
Comment (jsaini-hw) [inline scripts/travisPublishDocs.sh]: use `travis-ci-hyperwallet@paypal.com`

File: scripts/travisPublishDocs.sh [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+git config --global user.name "Travis"
```
Comment (jsaini-hw) [inline scripts/travisPublishDocs.sh]: use `travis-ci-hyperwallet`

File: scripts/travisPublishDocs.sh [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+# Publish docs
+mkdir ../gh-pages
+cp -r doc/_build/html/* ../gh-pages/
+cd ../gh-pages
+
+# Add branch
+git init
+git remote add origin https://${GH_TOKEN}@github.com/hyperwallet/python-sdk.git > /dev/null
```
Comment (jsaini-hw) [inline scripts/travisPublishDocs.sh]: Use `CI_USER_TOKEN` instead of `GH_TOKEN`

Comment (jsaini-hw) [reply scripts/travisPublishDocs.sh]: Update to hyperwallet-ios-ui-sdk.git instead of python-sdk.git

File: scripts/travisPublishDocs.sh [hunk 9]
```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+# Set identity
+git config --global user.email "travis-ci-hyperwallet@paypal.com"
+git config --global user.name "travis-ci-hyperwallet"
+
+# Publish docs
+mkdir ../gh-pages
+cp -r doc/_build/html/* ../gh-pages/
```
Comment (jsaini-hw) [inline scripts/travisPublishDocs.sh]: updated the docs folder path to what it should be in swift

File: .travis.yml (hunk 4 above)
Comment (sarun-altmtrk) [reply .travis.yml]: Added.

File: .travis.yml (hunk 5 above)
Comment (sarun-altmtrk) [reply .travis.yml]: Updated

File: scripts/travisPublishDocs.sh (hunk 6 above)
Comment (sarun-altmtrk) [reply scripts/travisPublishDocs.sh]: updated

File: scripts/travisPublishDocs.sh (hunk 7 above)
Comment (sarun-altmtrk) [reply scripts/travisPublishDocs.sh]: updated

File: scripts/travisPublishDocs.sh (hunk 8 above)
Comment (sarun-altmtrk) [reply scripts/travisPublishDocs.sh]: updated

File: scripts/travisPublishDocs.sh (hunk 9 above)
Comment (sarun-altmtrk) [reply scripts/travisPublishDocs.sh]: Updated

**Comment by jsaini-hw** [comment]
@sarun-altmtrk Kindly remove version and Docs from the title`HyperwalletSDK 0.0.1 Docs (53% documented)`

**Pull request merged**